.azurepipelines/appcenter-cli-1ES.yml

```diff
@@ -13,15 +17,24 @@ resources:
     name: 1ESPipelineTemplates/1ESPipelineTemplates
     ref: refs/tags/release
 
+parameters:
+- name: Skip1ESComplianceTasks
```

Comment:
I don't see a need to skip check - do you?
If not, please remove this parameter